modules/pipeline/dbclient/op_pipeline_task.go

```diff
@@ -267,8 +267,8 @@ func (client *Client) CleanPipelineTaskResult(id uint64, ops ...SessionOption) e
 	session := client.NewSession(ops...)
 	defer session.Close()
 
-	if _, err := session.Table("pipeline_tasks").ID(id).
-		Update(map[string]interface{}{"result": nil}); err != nil {
+	if _, err := session.Table("pipeline_tasks").
```

Comment:
judge id is 0

Reply:
don't need to judge zero, ID()-func need struct, map is invalid

Reply:
Defensive programming is needed. Someone maybe make a mistake here in the future.